addons/mail/static/src/new/core/messaging_service.js

```diff
@@ -536,15 +537,14 @@ export class Messaging {
                         this.store.threads[createLocalId("mail.channel", notif.payload.channel.id)];
                     const member = this.thread.insertChannelMember({
                         id: notif.payload.id,
-                        persona: this.partner.insertPersona({
-                            partner: this.partner.insert({
-                                id: notif.payload.persona.partner.id,
-                                name: notif.payload.persona.partner.name,
-                            }),
+                        persona: this.persona.insert({
+                            id: notif.payload.persona.partner.id,
+                            name: notif.payload.persona.partner.name,
+                            type: "partner",
                         }),
                         threadId: channel.id,
                     });
-                    if (member.persona.partner?.id === this.store.user.partnerId) {
+                    if (member.persona === this.store.self) {
```

Comment:
are we confident to compare records with reactive proxy?

Reply:
yes I am, everything is a proxy and getters return reference to proxy, so it's all good